### PR TITLE
Made fixers final

### LINKED ISSUE #1200: [2.0] Make Fixers Final
What do people think about making the fixer classes final on the master branch?

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Closes #1200.

### CODE REVIEW

File: Symfony/CS/Tests/FixerTest.php [hunk 1]
```diff
@@ -316,4 +316,26 @@ public function testCanFixWithConfigInterfaceImplementation()
...
+
+    public function provideFixersForFinalCheckCases()
+    {
+        $fixer = new Fixer();
+        $fixer->registerBuiltInFixers();
+        $fixers = $fixer->getFixers();
+        $cases = array();
+
+        foreach ($fixers as $fixer) {
+            $cases[] = array(new ReflectionClass($fixer));
```
Comment (keradus) [inline Symfony/CS/Tests/FixerTest.php]: `\ReflectionClass`

Comment (GrahamCampbell) [inline Symfony/CS/Tests/FixerTest.php]: Ahh yes, lol.

**Comment by keradus** [comment]
:+1:

**Comment by SpacePossum** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @GrahamCampbell.

**Pull request closed without merging**